submissions/chris-voitova/friends-app/script.js

```diff
@@ -0,0 +1,199 @@
+const url = "https://randomuser.me/api/?results=16";
+const usersContainer = document.querySelector(".users-container");
+
+const initState = {
```

Comment:
In one moment there can be only one selected input.

Why you need three variables for it?